settings.gradle

```diff
@@ -36,9 +36,9 @@ if (System.env.'COLLECTD_HOME') {
 //include 'distributions'
 
 
+include ':test:webapps:PerfTestApp'
```

Comment:
i thought this was sample app?

Reply:
oops, I didn't mean to include that...will revert. I was using that application for testing.